web/src/reactions.js

```diff
@@ -14,7 +14,9 @@ import * as people from "./people";
 import * as spectators from "./spectators";
 import {user_settings} from "./user_settings";
 
-export const view = {}; // function namespace
+export const view = {
+    loading_reactions_ids: {},
+}; // function namespace
```

Comment:
Should we use a `new Set()` here rather than an empty object?

Also, what do you think about calling this `waiting_for_server_reaction_ids`

Reply:
> Should we use a new Set() here rather than an empty object?

Sure, nice example of the place where Set object is appropriate and useful.

> Also, what do you think about calling this waiting_for_server_reaction_ids

It seems to me like a list of IDs that are waiting for a server reaction (response). However, I'm not entirely sure what these IDs are referring to. I think it would be more clear if the variable name explicitly mentioned that they are IDs for reactions. But, since the file is solely about reactions, maybe it's already clear enough. It's also possible that I may have misunderstood the meaning due to my language skills.

I think that "loading" is a well known term to indicate that a request is being made to a server and the application is waiting for a response and it's a good replacement for "waiting_for_server".

Reply:
I think `loading` is a good term when you're requesting data from the server, but feels off for a request where you're sending a change to the server and waiting for it to confirm receipt.

I went with `waiting_for_server_request_ids`, and also renamed the `message_reaction_id` local variable to `request_id`.